lib-multisrc/speedbinb/src/eu/kanade/tachiyomi/multisrc/speedbinb/SpeedBinb.kt

```diff
@@ -0,0 +1,289 @@
+package eu.kanade.tachiyomi.multisrc.speedbinb
+
+import eu.kanade.tachiyomi.lib.textinterceptor.TextInterceptor
+import eu.kanade.tachiyomi.lib.textinterceptor.TextInterceptorHelper
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Response
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import uy.kohesive.injekt.injectLazy
+import kotlin.math.ceil
+
+/**
+ * SpeedBinb is a reader for various Japanese manga sites. As it is **only** a reader,
+ * parsing of entries is left to the child class.
+ */
+abstract class SpeedBinb(
```

Comment:
```suggestion
abstract class SpeedBinbReader(
```
Make the name specific?

Reply:
Sure. b039b563